### PR TITLE
Quality/testing

### PR DESCRIPTION
**Pull request by ghost**
Adds:
- test cases and javadoc for baseLayout class
- test cases and javadoc for edge class
- graph mock

### CODE REVIEW

File: src/main/java/application/fxobjects/graph/cell/BaseLayout.java [hunk 1]
```diff
@@ -21,6 +21,9 @@
     private int maxDepth = 0;
     private int count = 0;
 
+    private static final int baseX = 200;
```
Comment (arbreurkes) [inline src/main/java/application/fxobjects/graph/cell/BaseLayout.java:24]: constants' names should be all capital letters.

File: src/main/java/application/fxobjects/graph/cell/BaseLayout.java [hunk 2]
```diff
@@ -21,6 +21,9 @@
...
+    private static final int baseY = 200;
```
Comment (arbreurkes) [inline src/main/java/application/fxobjects/graph/cell/BaseLayout.java:25]: constants' names should be all capital letters.

**Comment by arbreurkes** [comment]
The constant names in BaseLayout.java should be refactored to all caps. Can be merged now though.

**Comment by ghost** [comment]
the constant names in caps has already been done in a fix that is being worked on at the moment.

**Pull request merged**